backend/src/controller/chatController.ts

```diff
@@ -360,23 +356,26 @@ function handleGetChatHistory(req: OpenAiGetHistoryRequest, res: Response) {
 	}
 }
 
-function handleAddToChatHistory(req: OpenAiAddHistoryRequest, res: Response) {
-	const infoMessage = req.body.message;
+function handleAddInfoToChatHistory(
+	req: OpenAiAddInfoToChatHistoryRequest,
+	res: Response
+) {
```

Comment:
The more I think about this separate API call, the weirder it seems. In theory it allows us to inject an arbitrary message into the chat, of any chat message type. Ok we can't add a completion, but we could add a "BOT" message with an infoMessage property. Will the ChatGPT call simply ignore that message, as it has no completion property?

Anyhow, nothing to do here, just blurting out my thoughts as they form 😅

Reply:
well, not any more! They can only inject one that makes sense with the new checks I've added.
namely

```typescript
if (... && 
  chatInfoMessageType.includes(chatMessageType) &&
  ...
) {
  // do the stuff
}
```